test/mlir/onnx/onnx_lowering.mlir

```diff
@@ -2744,41 +2744,41 @@ func @test_split_unknown_dimension_equal_split(%arg0 : tensor<?x?x64xf32>) -> (t
   %0, %1 = "onnx.Split"(%arg0) { axis = 1 : si64 } : (tensor<?x?x64xf32>) -> (tensor<*xf32>, tensor<*xf32>)
   "std.return"(%0, %1) : (tensor<*xf32>, tensor<*xf32>) -> ()
 
-  // CHECK: [[INDEX_MAP:#.+]] = affine_map<(d0, d1) -> (d0 + d1 ceildiv 2)>
+  // CHECK: [[ALLOC_MAP0:#.+]] = affine_map<(d0, d1) -> (d0 ceildiv 2)>
```

Comment:
It is better to move the test to onnx_lowering_with_canonicalize as the map's math is moved into the code.

Reply:
Also, if you did the FileCheck format by hand, try my new `mlir2FileCheck` utility... it works quite well.

Reply:
`mlir2FileCheck` is quite useful. I used it. Thanks! However, it seemed it did not support local variables that have the same name, e.g. arguments of two krnl loops may use the same name. In the case of SplitOp, I manually changed variable names to make `mlir2FileCheck`work.